src/components/utils/CMakeLists.txt

```diff
@@ -28,244 +28,148 @@
-                ${UTILS_INCLUDE_DIR}/utils/gen_hash.h
-                ${UTILS_INCLUDE_DIR}/utils/helpers.h
-                ${UTILS_INCLUDE_DIR}/utils/signals.h
-                ${UTILS_INCLUDE_DIR}/utils/singleton.h
-                ${UTILS_INCLUDE_DIR}/utils/stl_utils.h
-                ${UTILS_INCLUDE_DIR}/utils/string_utils.h
-                ${UTILS_INCLUDE_DIR}/utils/system.h
-                ${UTILS_INCLUDE_DIR}/utils/json_utils.h
-                ${UTILS_INCLUDE_DIR}/utils/host_address.h
-                ${UTILS_INCLUDE_DIR}/utils/socket.h
-                ${UTILS_INCLUDE_DIR}/utils/socket_utils.h
-                ${UTILS_INCLUDE_DIR}/utils/timer.h
-                ${UTILS_INCLUDE_DIR}/utils/convert_utils.h)
-    set (src
-            ${UTILS_SRC_DIR}/bitstream.cc
-            ${UTILS_SRC_DIR}/conditional_variable_win.cc
-            ${UTILS_SRC_DIR}/date_time_win.cc
-            ${UTILS_SRC_DIR}/file_system_win.cc
-            ${UTILS_SRC_DIR}/gen_hash.cc
-            ${UTILS_SRC_DIR}/lock_win.cc
-            ${UTILS_SRC_DIR}/rwlock_win.cc
-            ${UTILS_SRC_DIR}/signals_win.cc
-            ${UTILS_SRC_DIR}/threads/thread_validator.cc
-            ${UTILS_SRC_DIR}/threads/async_runner.cc
-            ${UTILS_SRC_DIR}/threads/thread_delegate_win.cc
-            ${UTILS_SRC_DIR}/threads/win_thread.cc
-            ${UTILS_SRC_DIR}/string_utils.cc
-            ${UTILS_SRC_DIR}/pipe_win.cc
-            ${UTILS_SRC_DIR}/shared_library_win.cc
-            ${UTILS_SRC_DIR}/socket_win.cc
-            ${UTILS_SRC_DIR}/json_utils.cc
-            ${UTILS_SRC_DIR}/host_address.cc
-            ${UTILS_SRC_DIR}/socket_utils.cc
-            ${UTILS_SRC_DIR}/timer.cc
-            ${UTILS_SRC_DIR}/custom_string.cc
-            ${UTILS_SRC_DIR}/convert_utils.cc)
-  endif()
+set(LIBRARIES
+  config_profile
+  ${SQLITE3_LIBRARIES}
+)
 
-  if (BUILD_BT_SUPPORT)
-      list(APPEND inc
-               ${UTILS_INCLUDE_DIR}/utils/bluetooth_win/bluetooth_utils.h
-               ${UTILS_INCLUDE_DIR}/utils/bluetooth_win/bluetooth_uuid.h
-               ${UTILS_INCLUDE_DIR}/utils/bluetooth_win/bluetooth_service_record.h)
-      list(APPEND src
-         ${UTILS_SRC_DIR}/bluetooth_win/bluetooth_utils.cc)
-  endif()
-  if(ENABLE_LOG)
-    list(APPEND LIBRARIES config_profile)
-     if(QT_PORT)
-       list(APPEND src
-         ${UTILS_SRC_DIR}/logger_qt.cc
-         ${UTILS_SRC_DIR}/log_message_loop_thread_qt.cc
-       )
-     else()
-        list(APPEND src
-          ${UTILS_SRC_DIR}/logger_win.cc
-          ${UTILS_SRC_DIR}/log_message_loop_thread_win.cc
-        )
-     endif()
-  endif()
-    string( REPLACE / \\ group ${inc} )
-    source_group("Header Files" FILES ${inc} )
-    string( REPLACE / \\ group ${src} )
-    source_group("Source Files" FILES ${src} )
+get_sdk(SDK)
+if(SDK)
+  if(${SDK} STREQUAL "qt")
+    find_package(Qt5 REQUIRED Core)
+    find_package(Qt5 REQUIRED Network)
+    find_package(Qt5 REQUIRED Sql)
```

Comment:
@anosach-luxoft 
AFAIK, you can write that as find_package(Qt5 REQUIRED Core Network Sql)